Src/Generic/Styling/TextStyle.swift

```diff
@@ -117,4 +122,9 @@ extension TextStyle {
         style.font = HedvigFonts.circularStdBold!.withSize(17)
         style.color = .purple
     }
+    
+    static let drabbableOverlayTitle = TextStyle.default.restyled { (style: inout TextStyle) in
```

Comment:
Can we make the name more generic, maybe sync with Zak, he has an idea of what these are supposed to be called.

Reply:
Synced with Zak, turns out we're gonna have to change the look of the "header" of the draggable overlay a bit. I'll fix that.